README.adoc

```diff
@@ -4,49 +4,58 @@
 
 This repository is for drafts.
 
-Please submit your draft as early as possible. To submit:
+Please submit your draft as early as possible during the collection period.
+To submit:
 
-1. Fork the repository - reusing an old fork is OK.
-2. Create a new branch and enter the directory for the current time
-   period - `2022q3`.
-3. Create a new file, with a name that corresponds with the name of
-   your project - use the `report-sample.adoc` template.
+1. Fork the repository – reusing an old fork is OK.
+2. Create a new branch, and enter the directory for the current period – `2022q4`.
+3. Create a new file, with a name that corresponds with the name of your project – use the `report-sample.adoc` template.
 4. Create a pull request.
-5. Someone will review it and may request changes. If they do,
-   evaluate the requests, make any you think are warranted, explain
-   any you chose not to, and push again. *Do not squash or
-   force-push* if at all possible as this makes the next review
-   much harder.
+5. Someone will review the pull request.
+If changes are requested: evaluate them, make any that you think are warranted, explain any that you chose not to, and push again.
+** *Do not squash or force-push* if at all possible, as this makes the next review much harder.
 6. Once your draft is accepted and merged, you're good.
 
-You can tweak things after your draft was merged. Simply edit, then
-submit another PR.
+You can tweak things after your draft is merged.
+Simply edit, then submit another pull request.
 
-Please do not worry excessively about language errors or AsciiDoc
-syntax - we can correct things for you.
+Please do not worry excessively about language errors or AsciiDoc syntax – we can correct things for you.
 
-=== Deadline for submissions for `2022q3`
+=== Deadlines: general rules
 
-30th of September 2022
+Reports are collected during: 
```

Comment:
This should be consistent with what we are discussing above, so it should probably be changed by explaining that reports are collected and reviewed in any moment, but the main reviewing process starts at the beginning of the third month of each quarter.